### ISSUE TITLE
main-RANDOMHASH.css not being load when running `gulp --production`

### ISSUE THREAD

**Issue by mvaneijgen** [elsonponte/themename-child#1]
Everything works fine when running `gulp watch` but when the site is ready for production, the main-RANDOMHASH.css style are not injected into the head and only the parent styles are being loaded

**Comment by elsonponte** [comment]
Hi @mvaneijgen . Did you run gulp --production before deploying to production and uploading the child theme again?

**Comment by mvaneijgen** [comment]
Yes, like I also do with the parent Sage theme for normal productions. 

My workflow is:

`gulp` — usually there is something weird if you not first gulp
`gulp watch` — for development locally

`gulp` — usually there is something weird if you not first gulp
`gulp --production` — if im ready to deploy the site.

**Comment by elsonponte** [comment]
Try deleting the dist folder in the FTP and re-uploading the whole child theme again. Sometimes the browser does not load the new hashed CSS version because it has the previous one cached.

**Comment by mvaneijgen** [comment]
But the whole main-RANDOMHASH.css is missing from the head of my site. I can see it being generate in the dist folder, but when I open the source of my site, there is no (accept the parent main-RANDOMHAS.css file)

this is the parent
`<link rel="stylesheet" id="sage/css-css" href="http://domain.com/wp-content/themes/alloy-sage/dist/styles/main-2ed74986d8.css" type="text/css" media="all">`

and I also need the child to load 
`<link rel="stylesheet" id="sage/css-css" href="http://127.0.0.1/jam-japan/wp-content/themes/alloy-sage-CHILDTHEME/dist/styles/main-2ed74986d8.css" type="text/css" media="all">`

**Comment by elsonponte** [comment]
Are you uploading and overriding the full child theme again to the server?

**Comment by elsonponte** [comment]
Also, if you are using chrome, open a new private window or disable browser cache, as this may be the browser looking for the previous hashed file from its cache.

**Comment by elsonponte** [comment]
If that does not work, just run "gulp", delete the child theme from production and upload it again.

**Comment by mvaneijgen** [comment]
Cache is not an issue. The whole mytheme-CHILD main-HASH.css is not being enqueued. It is missing from the head section of the site. The style.css which only holds the WP data is being enqueued, but I would love to use SCSS on my site

**Comment by elsonponte** [comment]
Can you send me your website URL so I can check it?

**Comment by mvaneijgen** [comment]
@elsonponte not at the moment, but I have found the issue now. 

In my head there is the following line (if I run `gulp` or `gulp --production`)

```
<link rel='stylesheet' id='child-extra-style-css'  href='http:domain.com/wp-content/themes/alloy-sage-child/dist/styles/main.css?ver=4.8.6' type='text/css' media='all' />
```

but in my dist folder when I run `gulp --production` the file is called `main-58c87a579f.css` (or another random hash) which is not getting referenced. If I manually change `main-58c87a579f.css` to `main.css` everything works. 

So the question is how do I disable the random hash generation or how to I fix it so it will be referenced.

**Comment by elsonponte** [comment]
If you compile the assets with "gulp --production" you will automatically get the version of the files named. If you just run "gulp" you will notice the dist folder has the CSS as main.css without the versioning.

**Comment by elsonponte** [comment]
Just run gulp and "upload and override" the "whole child theme".
